modules/fedimint-wallet-common/src/lib.rs

```diff
@@ -358,6 +359,7 @@ pub struct WalletInputV0(pub Box<PegInProof>);
 pub struct WalletInputV1 {
     pub outpoint: bitcoin::OutPoint,
     pub tweak_contract_key: secp256k1::PublicKey,
+    pub script_pubkey: ScriptBuf,
```

Comment:
I guess by having some redundant data here we can make recovery easier? Might be worth it, but then we should check that the script pub key we submit is consistent with our DB when verifying inputs, so that later we don't accidentally depend on unverified data (when changing consensus in the future).

Reply:
Yes.